[2024-11-18T08:45:46.605+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_flow.transfrom_wikipedia_data manual__2024-11-18T08:45:25.036809+00:00 [queued]>
[2024-11-18T08:45:46.609+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_flow.transfrom_wikipedia_data manual__2024-11-18T08:45:25.036809+00:00 [queued]>
[2024-11-18T08:45:46.610+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-18T08:45:46.616+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): transfrom_wikipedia_data> on 2024-11-18 08:45:25.036809+00:00
[2024-11-18T08:45:46.618+0000] {standard_task_runner.py:57} INFO - Started process 1613 to run task
[2024-11-18T08:45:46.620+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_flow', 'transfrom_wikipedia_data', 'manual__2024-11-18T08:45:25.036809+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia.py', '--cfg-path', '/tmp/tmpla5jcjx5']
[2024-11-18T08:45:46.622+0000] {standard_task_runner.py:85} INFO - Job 29: Subtask transfrom_wikipedia_data
[2024-11-18T08:45:46.650+0000] {task_command.py:415} INFO - Running <TaskInstance: wikipedia_flow.transfrom_wikipedia_data manual__2024-11-18T08:45:25.036809+00:00 [running]> on host scheduler
[2024-11-18T08:45:46.765+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Clarkson' AIRFLOW_CTX_DAG_ID='wikipedia_flow' AIRFLOW_CTX_TASK_ID='transfrom_wikipedia_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T08:45:25.036809+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-18T08:45:25.036809+00:00'
[2024-11-18T08:49:13.591+0000] {local_task_job_runner.py:115} ERROR - Received SIGTERM. Terminating subprocesses
[2024-11-18T08:49:13.596+0000] {process_utils.py:131} INFO - Sending 15 to group 1613. PIDs of all processes in the group: [1613]
[2024-11-18T08:49:13.597+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 1613
[2024-11-18T08:49:13.598+0000] {taskinstance.py:1630} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-11-18T08:49:13.600+0000] {logging_mixin.py:151} INFO - Error while geocoding BORUSSIA-PARK, Germany: Task received SIGTERM signal
[2024-11-18T08:49:16.292+0000] {taskinstance.py:1147} INFO - Dependencies not met for <TaskInstance: wikipedia_flow.transfrom_wikipedia_data manual__2024-11-18T08:45:25.036809+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2024-11-18T08:49:16.295+0000] {taskinstance.py:1147} INFO - Dependencies not met for <TaskInstance: wikipedia_flow.transfrom_wikipedia_data manual__2024-11-18T08:45:25.036809+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2024-11-18T08:49:16.296+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
